.github/workflows/frontend_plugin_test.yml

```diff
@@ -51,10 +51,16 @@ jobs:
         backend: 'true'
         frontend: 'true'
         indexer: 'true'
+      
+    - name: Setup firefox
```

Comment:
Is this testing for Firefox compatibility against our plugins? Does it pull the latest version we support here at SI?

Reply:
No, in order to run frontend tests we need a (heady or headless) browser to run inside our Github actions.  Here we're just saying to use this standard setup-firefox action to accomplish this: https://github.com/browser-actions/setup-firefox

If you were running tests locally on your own machine, you'd accomplish the same by having Firefox (or Chrome or whatever) installed on your path with the appropriate drivers.  We need to do that in our actions environment as well for the frontend tests to run.  We didn't need this before because our only existing frontend test was was a controller test (frontend/spec/archival_objects_controller_ext_spec.rb) and it didn't require a full browser.  This new feature spec does require a browser.